### ISSUE TITLE
Traceback when Atom config path contains spaces

### ISSUE THREAD

**Issue by Rylon** [h-hirokawa/atom-autocomplete-ansible#14]
Hi, I'm receiving the following error with autocomplete-ansible:

```
autocomplete-python traceback output:

Command failed: /usr/local/bin/python /Users/ryan/Library/Mobile Documents/com~apple~CloudDocs/Sync/AtomEditor/packages/autocomplete-ansible/lib/parse_ansible.py
/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file '/Users/ryan/Library/Mobile': [Errno 2] No such file or directory
```

The `/Users/ryan/Library/Mobile Documents/com~apple~CloudDocs/Sync/AtomEditor/` path is where my Atom configuration is stored, with a symlink pointing to it from `~/.atom`, so that my configuration can be shared between multiple computers. The path in question here is the system location of the macOS "iCloud Drive", and can not be changed.

You can see that the "Mobile Documents" part of the path contains a space, which seems to be confusing autocomplete-ansible or maybe autocomplete+ itself?

Thanks!

**Comment by Rylon** [comment]
I have a quick look through the code, and I found a potential issue on line 36 of this file: https://github.com/h-hirokawa/atom-autocomplete-ansible/blob/master/lib/provider.js#L36

I'm not sure of the syntax there, but would quoting the filename help? For example:

```
exec(`${interpreter} "${__dirname}/parse_ansible.py"`, {maxBuffer: 5*1024*1024}, function (err, stdout) {
```

**Comment by h-hirokawa** [comment]
@Rylon 
Thank you for reporting the issue.
The fix is released under v0.1.11.

**Comment by Rylon** [comment]
No problem, thanks for the quick fix @h-hirokawa 👍
